Gaijinizer patch/Map054.txt

```diff
@@ -980,75 +980,80 @@ disaster Ne, \\N[1].#MTLed
 だいぶ女悪魔の扱いに慣れてきたんじゃない？、
 女悪魔キラーの\\N[1]サン♪
 > CONTEXT: Map054/events/1/pages/5/21/Dialogue/hyakki
-Even though it is a lower-grade monster, it is
-important not to move to the temptation of that
-Te. Did you get used to the treatment of demon
-for a while? , Woman demon killer \\N[1] San♪#MTLed
+Even if they're low-rank youkai, that doesn't
+mean their temptation attacks won't affect you.
+Haven't you gotten used to dealing with female
+demons? Lady Demon Killer \\N[1]-San♪#Strate
 > END STRING
 
 > BEGIN STRING
 んで、そんなレディ・キラーなジゴロさんに忠告なんだけど…
 
 ちょっと今度マイルーム行くときは注意した方がいいかも…
 > CONTEXT: Map054/events/1/pages/5/29/Dialogue/hyakki
-Well, I'm advising such a lady killer Gigolo ...
-I should be careful when I go to my room for a
-moment ...#MTLed
+So, a little advice for Ladykiller Gigolo-san...
+Perhaps you should be careful next time you
+go to My Room...#Strate
 > END STRING
 
 > BEGIN STRING
 え、どういうこと…？
 > CONTEXT: Map054/events/1/pages/5/34/Dialogue/kazuya
-Eh, what kind of thing ...?#MTLed
+Eh, what do you mean...?#Strate
 > END STRING
 
 > BEGIN STRING
 じ、実はね……
 
 \\}ルシィさん…帰ってなかったの…
 > CONTEXT: Map054/events/1/pages/5/36/Dialogue/hyakki
-Really...\\}Lucy...I did not return ...#MTLed
+T-the truth is...
+
+\\}Lucy-san...didn't go back...#Strate
 > END STRING
 
 > BEGIN STRING
 \\}そ、それで……
 
 あの…、君とあの子達との…その…『現場』を……
 > CONTEXT: Map054/events/1/pages/5/40/Dialogue/hyakki
-\\}Well then...That ..., with you and his children ...
-that..."the scene" ...#MTLed
+\\}S-so......
+That..., with you and those kids...
+that..."scene"...#StrateUnsure
 > END STRING
 
 > BEGIN STRING
 ルシィさん、\\N[1]が抱きかかえられたあたりで
 
 『そういうことですか…』って呟いて帰っちゃった……
 > CONTEXT: Map054/events/1/pages/5/48/Dialogue/hyakki
-Lucy, around \\N[1] is Hug. "What's that ...?"#MTLed
+Around the time you were being hugged, Lucy-san
+muttered "So that's how it is..." and left...#Strate
 > END STRING
 
 > BEGIN STRING
 だからさ……気をつけなよ……
 
 あの人、静かに怒るタイプだし、どんな罰考えてるか…
 > CONTEXT: Map054/events/1/pages/5/52/Dialogue/hyakki
-That's why...be careful...
-That person is the type to get angry quietly,
-what punishment do you think ...#MTLed
+So...be careful...
+
+That person, she's the type to get angry quietly,
+thinking of what punishment to give...#StrateLocalization
 > END STRING
 
 > BEGIN STRING
 ………なんか、ずっとこの裏路地で生活したくなってきた…。
 > CONTEXT: Map054/events/1/pages/5/57/Dialogue/kazuya
-......... I have wanted to live in this back
-alley all the time ....#MTLed
+.........Somehow, I suddenly really feel like
+living in this back alley now...#StrateLocalization
 > END STRING
```

Comment:
Some changes to the ellipses, and a better rewording.
>_**......For some reason, I suddenly feel like
living in this back alley now...**_